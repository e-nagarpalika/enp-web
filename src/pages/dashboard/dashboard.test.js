/** @format */

import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import Dashboard from "./dashboard";

test("renders a message", () => {
  const { container } = render(<Dashboard />);

  expect(container).toMatchSnapshot();
});
