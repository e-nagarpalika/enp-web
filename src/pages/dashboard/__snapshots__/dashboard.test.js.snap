// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<div>
  <div
    style="text-align: center;"
  >
    <div>
      Dashboard
    </div>
    <button
      style="height: 40px; width: 100px; align-items: flex;"
    >
      Sign Out
    </button>
  </div>
</div>
`;
